### PR TITLE
Added Jetpack Glance Theming 

### PR DESCRIPTION
**Pull request by tuuhin**
## Added Colors to App Widget
App Widget is now responsive to the dynamic colors of the app on Android 12+,on lower version the app light and dark theme is used.

### Secondary Changes 
- Rounded shape added on version lesser than   Android 12 
- Added no task drawable to give a pictorial representation of no tasks

### CODE REVIEW

**Comment by tuuhin** [comment]
Don't understand why the build failed, the reason of failure shown in the workflow, it happened due to something related to keystore.
Can someone explain ??

**Comment by tuuhin** [comment]
It's seems due to the keytool the build is failing.Its showing keystore  was tampered with or password was incorrect.Could you elaborate on this.(why the build is failing)

**Comment by igorescodro** [comment]
> It's seems due to the keytool the build is failing.Its showing keystore was tampered with or password was incorrect.Could you elaborate on this.(why the build is failing)

The current CI structure is not very fork friendly. The `build.yml` generates the signed `release` variant to make sure that the app is production ready. However, all the keys and passwords are GitHub Secrets, and GitHub doesn't provide this information for non-contributors.

I'll try to update the CI to run with fake credentials when an external PR is submitted, but this might take a while. In the meantime, I recommend you run `./gradlew check test` locally to make sure your code is in compliance with our linters. Thank you for your contribution.

**Comment by igorescodro** [comment]
Awesome! Now it’s working properly. The CI is failing due to lint checks. I believe you are good to go.

Unfortunately, the Instrumented Tests are very flaky recently so I might need to rerun the a few times.

**Comment by tuuhin** [comment]
Great to hear that the workflows are working fine now!

File: features/glance/src/main/java/com/escodro/glance/presentation/TaskListGlanceWidget.kt [hunk 1]
```diff
@@ -27,114 +28,149 @@ import androidx.glance.appwidget.provideContent
...
+        context: Context = LocalContext.current,
+    ) {
+        Column(
+            modifier = modifier.fillMaxSize(),
+            verticalAlignment = Alignment.CenterVertically,
+            horizontalAlignment = Alignment.CenterHorizontally,
+        ) {
+            Image(
+                provider = ImageProvider(R.drawable.ic_task_list),
+                contentDescription = "Blank Task List",
```
Comment (igorescodro) [inline features/glance/src/main/java/com/escodro/glance/presentation/TaskListGlanceWidget.kt:119]: Please, add this string to the `strings.xml`. The Portuguese translation is "Lista de Tarefas Vazia".

**Review by igorescodro — COMMENTED** [review]
Thanks for your contribution! Just run the app and the new widget looks great!

**Pull request merged**